MobileTraining/MobileTraining/Networking/Repositories/LoginRepository.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PostRepository.swift
+//  MobileTraining
+//
+//  Created by Svetoslav Donkov on 16.05.23.
+//
+
+import Foundation
+
+class LoginRepository: LoginProtocol {
+    private let networkManager = NetworkManager()
+    
+    func fetchLogin(identifier: String, password: String) async throws -> Login {
+        let url = "https://ethereal-artefacts.fly.dev/api/auth/local"
```

Comment:
Extract the BASE URL to a constant in a `Constants` file or even better - to a plist configuration env variable.